app.py

```diff
@@ -370,6 +394,7 @@ def additional_projections_chart(i: np.ndarray, r: np.ndarray) -> alt.Chart:
     """* **Hospitalized COVID-19 Patients:** The number of patients currently hospitalized with COVID-19. This number is used in conjunction with Hospital Market Share and Hospitalization % to estimate the total number of infected individuals in your region.
 * **Currently Known Regional Infections**: The number of infections reported in your hospital's catchment region. This input is used to estimate the detection rate of infected individuals. 
 * **Doubling Time (days):** This parameter drives the rate of new cases during the early phases of the outbreak. The American Hospital Association currently projects doubling rates between 7 and 10 days. The doubling time will be higher (slower spread) or lower (faster spread) depending on the amount of social distancing in your region. 
```

Comment:
Not sure how to re-write this, but we should change it to make it clear that this should be the estimated doubling time **in the absence** of any social distancing measures.